spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -223,80 +247,6 @@
     end
   end
 
-  describe "#assign_visibility?" do
-    context "when no params are specified" do
-      it "does not need to assign visibility" do
-        expect(actor.send(:assign_visibility?)).to eq false
-      end
-    end
-
-    context "when file set params with visibility are specified with symbols as keys" do
-      let(:file_set_params) { { visibility: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC } }
-
-      it "does need to assign visibility" do
-        expect(actor.send(:assign_visibility?, file_set_params)).to eq true
-      end
-    end
-
-    context "when file set params with visibility are specified with strings as keys" do
-      let(:file_set_params) { { "visibility" => Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC } }
-
-      it "does need to assign visibility" do
-        expect(actor.send(:assign_visibility?, file_set_params)).to eq true
-      end
-    end
-  end
-
-  describe "#set_representative" do
```

Comment:
Why was this test coverage removed? Previously we covered thumbnail set and unset. By removing this method, which was unnecessary to acchieving the JobIoWrapper, you've made the code less testable.  I'd like to see these tests and their methods restored.